### PR TITLE
Simplify library releasing

### PR DESCRIPTION
**Pull request by maxokorokov**
This aims to simplify and a bit better automate library releasing by:
- freezing yarn version used
- using `yarn version` for version update
- generating ng-bootstrap package version

There are 3 separate commits for each part

For reference → [current release process is here](https://github.com/ng-bootstrap/ng-bootstrap/wiki/Contributions%3A-Release)

New release process would look like this:

## Version X.X.X + publish

* [ ] make sure you're on the `master` branch and you're ready to release
* [ ] generate the new release (`yarn version`). This will generate a tagged release commit with changelog updates, build the library and the demo site.
* [ ] check the `CHANGELOG.md` in the last commit. If necessary update the file and run `yarn:changelog-patch` to amend changelog and update the git release tag.
* [ ] push to npm (`cd dist/ng-bootstrap`, `npm publish`, `cd ../..`). There is no way of NOT rewriting the `latest` tag in npm, so when publishing OLDER releases, you have to manually restore the `latest` tag with `npm dist-tags add @ng-bootstrap/ng-bootstrap@Z.Z.Z latest`
* [ ] push `master` to GitHub (`git push upstream`)
* [ ] push tags to GitHub (`git push upstream --tags`)
* [ ] push the demo site (`yarn demo:publish`)
* [ ] close the release [milestone](https://github.com/ng-bootstrap/ng-bootstrap/milestones)
* [ ] let the World know!

Will update the document when this is merged

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +20,7 @@
     "demo:ie": "yarn demo -c ie",
     "ssr": "yarn ssr-app:lint && yarn ssr-app:build && yarn ssr-app:e2e",
     "changelog": "conventional-changelog --preset angular --infile CHANGELOG.md --same-file --release-count 1",
+    "changelog:patch": "git add CHANGELOG.md && git commit --amend --no-edit && git tag -f $npm_package_version",
```
Comment (benouat) [inline package.json:23]: Is `$npm_package_version` automatically there thanks to nom cli ?

Comment (maxokorokov) [reply package.json:23]: yeah: https://docs.npmjs.com/misc/scripts#packagejson-vars

The only issue is that on Windows one should use `%$npm_package_version%

Comment (benouat) [reply package.json:23]: Even on WSL or Cygwin ? `% ... %` sounds more like CMD or PowerShell

Comment (maxokorokov) [reply package.json:23]: Yes, I think you're right

**Pull request merged**